Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockGamepadPS4.cs

```diff
@@ -319,6 +470,13 @@ public override void SetMotorSpeeds(float largeMotor, float smallMotor)
             m_LargeMotor = largeMotor;
             m_SmallMotor = smallMotor;
         }
+        public void ResetOrientation()
```

Comment:
Will this reset just the orientation reading or the entire gyro data?

Reply:
Only resets the orientation. The other two, velocity and angular velocity, are accelerometers. Although the documentation says angular velocity gets recalculated from the new orientation it won't get reset to 0's. The native method is called scePadResetOrientation which only sets the orientation to an identity quaternion (0,0,0,1).